libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -32,6 +32,8 @@ func newEc2MetadataFetcher(config *common.Config) (*metadataFetcher, error) {
 			"machine_type":      c.Str("instanceType"),
 			"region":            c.Str("region"),
 			"availability_zone": c.Str("availabilityZone"),
+			"accountId":         c.Str("accountId"),
```

Comment:
This should use `account.id` to align with ECS.

Reply:
@andrewkroh 
do you mean `cloud.account.id`?

I guess some other fields in there could also fit the cloud ecs schema
https://www.elastic.co/guide/en/beats/filebeat/current/add-cloud-metadata.html

(looks that this task is also needed for most other cloud metadata processors

Reply:
The `cloud.` part gets added later. So changing the left side `accoundId` to `account.id` should make this match with the ECS `cloud.account.id` field.

Reply:
@odacremolbap maybe https://www.elastic.co/guide/en/ecs/current/ecs-cloud.html will help!

Reply:
I meant there are some other fields that looks not ECS compliant, like `machine_type` (should be `machine.type`), `instance_id`(should be `instance.id`)

am I missing something there?
I'll go through all cloud metadata processors, since I think there are a number of non ECS compliant schemas